iib/web/static/api_v1.yaml

```diff
@@ -0,0 +1,461 @@
+components:
+  schemas:
+    BundleMapping:
+      type: object
+      example:
+        some_operator:
+          - registry.com/op/go-logging
+      properties:
+        operator:
+          type: array
+          items:
+            type: string
+          example:
+            - some_bundle
+            - some_other_bundle
+    Pagination:
+      type: object
+      properties:
+        first:
+          type: string
+          example: >-
+            https://iib.domain.local/api/v1/builds?page=1&per_page=20&verbose=False
+        last:
+          type: string
+          example: >-
+            https://iib.domain.local/api/v1/builds?page=3&per_page=20&verbose=False
+        next:
+          type: string
+          example: >-
+            https://iib.domain.local/api/v1/builds?page=1&per_page=20&verbose=False
+        page:
+          type: integer
+          example: 1
+        pages:
+          type: integer
+          example: 3
+        per_page:
+          type: integer
+          example: 20
+        previous:
+          type: string
+          example: null
+        total:
+          type: integer
+          example: 45
+    Request:
+      type: object
+      properties:
+        id:
+          type: integer
+          example: 1
+        arches:
+          type: array
+          items:
+            type: string
+          example:
+            - amd64
+            - s390x
+        binary_image:
+          type: string
+          example: 'quay.io/operator-framework/upstream-registry-builder:v1.5.9'
+        binary_image_resolved:
+          type: string
+          example: >-
+            quay.io/operator-framework/upstream-registry-builder@sha256:7d8e5dddad0275bc903b6ef17840f98441b15b8b999609af2c9579960e52080e
+        bundles:
+          type: array
+          items:
+            type: string
+          example:
+            - 'quay.io/user/openshift-bundle-example-container:v1'
+        bundle_mapping:
+          $ref: '#/components/schemas/BundleMapping'
+        from_index:
+          type: string
+          example: 'quay.io/iib-stage/iib:4'
+        from_index_resolved:
+          type: string
+          example: >-
+            quay.io/iib-stage/iib@sha256:7d8e5dddad0275bc903b6ef17840f98441b15b8b999609af2c9579960e52080e
+        organization:
+          type: string
+          example: openshift-operators
+        removed_operators:
+          type: array
+          items:
+            type: string
+          example:
+            - prometheus-operator
+        request_type:
+          type: string
+          example: add
+        state:
+          type: string
+          example: complete
+        state_reason:
+          type: string
+          example: Completed successfully
+        updated:
+          type: string
+          example: '2019-09-17T19:42:47.149979'
+        user:
+          type: string
+          example: msdhoni@DOMAIN.LOCAL
+    RequestAdd:
+      type: object
+      properties:
+        add_arches:
```

Comment:
Could you add a description explaining that if this is not provided, `from_index` must be provided?